### ISSUE TITLE
"Download" button present in 'Response body' code is not accessible using keyboard keys. :A11y_VS Code extension for API Center_OpenAPIDocumentation_CodeExecute_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/vscode-azureapicenter#114]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG2.1.1; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #Linux; #FTP;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create a API center in azure portal. Open Visual studio Code.
2. Tab till API center extension in the left side panel and press enter.
3. Tab till definition under the api name which you have created and press enter.
4. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
5. Go to "Open API Documentation" using the down arrow and press enter.
6. Demo conference API screen will appear.
7. Tab till sessions dropdown button and press enter. Tab till try it out and press enter.
8. Tab till execute and press enter.
9. Navigate using TAB key to "Download" button present in 'Response body' code and observe the issue.

### Actual Result:
"Download" button present in 'Response body' code is not accessible using keyboard TAB keys.

**Note: Button present in 'Curl' is also not accessible using keyboard and mouse.**

### Expected Result:
"Download" button present in 'Response body' code should be accessible using keyboard TAB keys.

### User Impact:
Keyboard only users will not be able to access the download button and will not be able to perform the further action.

### Attachment:
![114_A11y_VS Code extension for API Center_OpenAPIDocumentation_CodeExecute_Keyboard](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/ef2084c1-3649-4bd9-8990-00dcab602af1)

**Comment by formulahendry** [comment]
@PoojaNamde Could you please break down the two issues into two different bugs?

**Comment by PoojaNamde** [comment]
Created new bug for another issue https://github.com/microsoft/vscode-azureapicenter/issues/138
